reference/robot.md

```diff
@@ -65,7 +65,8 @@ two remain constant. *Important:* when the current energy value reaches zero,
 the corresponding controller process terminates and the simulated robot stops
 all motion.
 
-    Note: *[J]=[V].[A].[s] and [J]=[V].[A.h]/3600*
+> **note**:
+*[J]=[V].[A].[s] and [J]=[V].[A.h]/3600*
```

Comment:
No sure because just above in the text we use italics for similar equations:  (*[W]=[J]/[s]*)